clients/lsp-xml.el

```diff
@@ -287,13 +287,20 @@ The value for `enabled' can be always, never or onValidSchema."
   ("xml.catalogs" lsp-xml-catalogs)
   ("xml.trace.server" lsp-xml-trace-server)))
 
+(defconst lsp-xml-has-java? (executable-find "java"))
```

Comment:
Can we move it elsewhere so it doesn't get executed at the root level? 🤔

Reply:
Good point.